scripts/artifacthub/go.mod

```diff
@@ -0,0 +1,11 @@
+module artifacthub
```

Comment:
should we have a single go.mod

Reply:
I purposely kept two (and any future) modules separate so that we don't have to unnecessarily add dependencies in one due to another. Let me know if I am missing anything by not combining them.